src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -34,10 +34,6 @@ Released 2024-Sep-30
   on mobile platforms which caused telemetry to be dropped silently.
   ([#5821](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/5821))
 
-* Updated `Microsoft.Extensions.Hosting.Abstractions` package
-  version to `9.0.0-rc.1.24431.7`.
-  ([#5853](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5853))
```

Comment:
@rajkumar-rangaraj This was added here: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5853/files#diff-1788690fb9e80d48d96d92320b51dc391560eb447490867db722279f47939d1eR25-R27

But OTLP doesn't have a dependency on `Microsoft.Extensions.Hosting.Abstractions`. Seems like a copy/paste issue? I just went ahead and removed it.